squirtlesquad2021a/pokedeck_capstone_app
# Trying out more tests

added some tests to make sure UI elements were rendering.

Steps to Test:
•clone the repo
•git fetch --all
•git checkout trying-out-more-tests
•git pull origin trying-out-more-tests
•yarn test

(all tests should pass)
